src/_incydr_sdk/exceptions.py

```diff
@@ -1,9 +1,29 @@
+from pydantic import ValidationError
+
+
 class IncydrException(Exception):
     """Base class for all Incydr specific exceptions."""
 
     ...
 
 
+class AuthMissingError(ValidationError, IncydrException):
+    def __init__(self, validation_error: ValidationError):
+        self.pydantic_error = str(validation_error)
+        super().__init__(validation_error.raw_errors, validation_error.model)
+
+    @property
+    def error_keys(self):
+        return [e["loc"][0] for e in self.errors()]
```

Comment:
Do we know that these keys will always be present? Ie. theres no chance of a `KeyError` or anything?

Reply:
In this case, I believe so. Since we're only using this when catching the `pydantic.ValidationError` specifically on the `IncydrSettings` model, there _shouldn't_ be any weird validation edge cases to worry about. 🤞